plugins/mission/mission_item.h

```diff
@@ -87,6 +89,13 @@ class MissionItem
         NONE /**< @brief No action. */
     };
 
+    /**
+     * @brief Converts #CameraAction to English strings.
```

Comment:
Why is it #CameraAction ?

Reply:
Doxygen tag/command for [linking other members](https://www.stack.nl/~dimitri/doxygen/manual/autolink.html#linkother).

Reply:
That would actually link to the constructor according to your docs - which I doubt is what you want. You don't need to mark it up at all to link to CameraAction - Doxygen does that automatically. More often I want to stop it linking :-) (and use `%`)